portal-web/test/functional/com/liferay/portalweb/macros/journal/template/WebContentStructures.macro

```diff
@@ -111,6 +111,12 @@ definition {
 		}
 	}
 
+	macro editStructureCP {
```

Comment:
Please, don't use/add this macro. There is a macro available called editStructure .